civihr_employee_portal/features/civihr_default_permissions/civihr_default_permissions.features.user_permission.inc

```diff
@@ -2183,19 +2183,6 @@ function civihr_default_permissions_user_default_permissions() {
     'module' => 'civicrm',
   );
 
-  // Exported permission: 'view appraisals'.
-  $permissions['view appraisals'] = array(
-    'name' => 'view appraisals',
-    'roles' => array(
-      'HR Admin' => 'HR Admin',
-      'Manager' => 'Manager',
-      'Regional HR Admin' => 'Regional HR Admin',
-      'Staff' => 'Staff',
-      'administrator' => 'administrator',
-    ),
-    'module' => 'civihr_employee_portal',
-  );
```

Comment:
You'll also need to add an upgrader to delete these permissions from the database and another one to revert the default permissions feature